aviary/docs/getting_started/onboarding_level3.ipynb

```diff
@@ -67,14 +67,16 @@
     "In `aviary/validation_cases/benchmark_tests` folder, there is an N3CC aircraft full mission benchmark test `test_FLOPS_based_sizing_N3CC.py`. Now, we will show how to create an N3CC example in level3. The key is that we follow the same steps:\n",
     "- {glue:md}`__init__()`\n",
     "- {glue:md}`load_inputs()`\n",
-    "- {glue:md}`check_and_preprocess_inputs()` (optional)\n",
+    "- {glue:md}`prob.check_and_preprocess_inputs()`\n",
```

Comment:
New additiona have a `prob.` in front of them, which isn't consistent with the rest.